### PR TITLE
OSDOCS-5507: Moving up versions for 4.13

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13 only

Issue:
https://issues.redhat.com/browse/OSDOCS-5507

Link to docs preview:
https://57170--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html
https://57170--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#installing-microshift-from-rpm-package_microshift-install-rpm (repo paths)
https://57170--docspreview.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree.html (repo paths)

QE review:
-SME approval (see lgtm pre-rebase)

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@fzdarsky PTAL at the updated repo paths and version info in the Install sections for 4.13.
@jogeo PTAL and advise that the versions as tested by QE are correct.

File: modules/microshift-adding-repos-to-image-builder.adoc [hunk 1]
```diff
@@ -37,8 +37,8 @@ $ sudo yum install -y yum-utils
 ----
 $ sudo reposync --arch=$(uname -i) --arch=noarch --gpgcheck \
     --download-path REPO_PATH=/var/repos/microshift-local \
-    --repo=rhocp-{product-version}-for-rhel-{op-system-version-major}-$(uname -i)-rpms \
```
Comment (fzdarsky) [inline modules/microshift-adding-repos-to-image-builder.adoc:40]: This needs replacing `{product-version}` to `{ocp-version}`, too.

File: modules/microshift-adding-repos-to-image-builder.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ Use the following procedure to add the {product-title} repositories to Image Bui
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos
-    --enable rhocp-{product-version}-for-rhel-{op-system-version-major}-$(uname -i)-rpms
+    --enable rhocp-{ocp-version}-for-rhel-{rhel-major}-$(uname -i)-rpms
```
Comment (fzdarsky) [inline modules/microshift-adding-repos-to-image-builder.adoc:23]: `rhel-{rhel-major}` --> `{rhel-major}`

**Review by fzdarsky — CHANGES_REQUESTED** [review]

File: microshift_install/microshift-embed-in-rpm-ostree.adoc [hunk 3]
```diff
@@ -6,19 +6,20 @@ include::_attributes/attributes-microshift.adoc[]
...
-You can embed {product-title} into a {op-system-ostree-first} {op-system-version} image. Use the content within this section to build a {op-system} image containing {product-title}.
+You can embed {product-title} into a {op-system-ostree-first} {op-system-version} image. Use this guide to build a {op-system} image containing {product-title}.
 
 include::snippets/microshift-tech-preview-snip.adoc[leveloffset=+1]
 
 [id="preparing-for-image-building_{context}"]
 == Preparing for image building
 
-Familiarize yourself with the documentation about https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images[Composing, installing, and managing RHEL for Edge images].
+Read https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images[Composing, installing, and managing RHEL for Edge images].
```
Comment (dhellmann) [inline microshift_install/microshift-embed-in-rpm-ostree.adoc:16]: Why does this change from 9 to 8?

**Review by dhellmann — COMMENTED** [review]
One question, but otherwise

/lgtm

File: microshift_install/microshift-embed-in-rpm-ostree.adoc (hunk 3 above)
Comment (ShaunaDiaz) [reply microshift_install/microshift-embed-in-rpm-ostree.adoc:16]: Weird, it shouldn't. We had a problem with a cherry-pick yesterday, the repos were out of sync (extra commit somewhere). Shall fix.

**Comment by jboxman-rh** [comment]
@ShaunaDiaz, LGTM, thanks!

**Comment by jboxman-rh** [comment]
@ShaunaDiaz, that's odd, shouldn't this merge into 4.13 cleanly? I'm not familiar with how microshift differs in this.

**Comment by ShaunaDiaz** [comment]
@jboxman-rh It should. We had a problem yesterday. @kalexand-rh WDYT? Manual CP or something else needed?

**Pull request merged**